scripts/fix-missing-project-id.sh

```
@@ -0,0 +1,119 @@
+        description VARCHAR(256) NOT NULL DEFAULT '' COMMENT 'Project description',
+        color VARCHAR(20) NOT NULL DEFAULT '' COMMENT 'Project color identifier (e.g., #FF5733)',
+        sort_order INT NOT NULL DEFAULT 0 COMMENT 'Sort order for display',
+        is_expanded TINYINT(1) NOT NULL DEFAULT 1 COMMENT 'Whether the project is expanded in UI',
+        is_active TINYINT(1) NOT NULL DEFAULT 1 COMMENT 'Whether the project is active (soft delete)',
+        created_at DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP COMMENT 'Creation timestamp',
+        updated_at DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT 'Last update timestamp',
+        PRIMARY KEY (id),
+        KEY idx_projects_user_id (user_id),
+        KEY idx_projects_sort_order (sort_order)
+    ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COMMENT='Projects table for task organization';
+    " 2>&1 > /dev/null
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)